src/wp-includes/media.php

```diff
@@ -1857,6 +1857,10 @@ function wp_filter_content_tags( $content, $context = null ) {
 			if ( $filtered_image !== $match[0] ) {
 				$content = str_replace( $match[0], $filtered_image, $content );
 			}
+
+			// Unset image lookup to not run the same logic again unnecessarily if the same image tag is used more than
+			// once in the same blob of content.
```

Comment:
```suggestion
			/*
			 * Unset image lookup to not run the same logic again unnecessarily if the same image tag is used more than
			 * once in the same blob of content.
			 */
```

Nitpick: Multiline commenting standards